lib/src/components/NxToast/NxToast.scss

```diff
@@ -5,28 +5,21 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 
-.nx-toast-wrapper {
-  // nx-toast-wrapper has position absolute to ensure toasts stay in the top right corner of the page.
-  // Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
-  // nx-page-main. The nx-toast-container has position fixed applied so it remains fixed while scrolling.
-  position: absolute;
-  top: 0;
-  right: 0;
-  display: flex;
-  justify-content: flex-end;
-  margin-right: var(--nx-spacing-6x);
-}
-
 .nx-toast-container {
-  position: fixed;
+  position: sticky;
+  left: calc(100% - 500px);
```

Comment:
`right: 0;` should do the same thing and be simpler

Reply:
> `right: 0;` should do the same thing and be simpler

I thought so too, but unfortunately doesn't affect the positioning, which is why I tried left. I played around with aligning items with flex, but that requires the width of `nx-toast-container` to be 100%, and then affects being able to click elements on the page due to the z-index that needs to be applied to put it to the top of the page.

Reply:
> I thought so too, but unfortunately doesn't affect the positioning, which is why I tried left. I played around with aligning items with flex, but that requires the width of nx-toast-container to be 100%, and then affects being able to click elements on the page due to the z-index that needs to be applied to put it to the top of the page.

https://github.com/sonatype/sonatype-react-shared-components/pull/688#discussion_r956105301

Reply:
> right: 0; should do the same thing and be simpler

resolved here with justify-self: [5ee0198](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/5ee019891c34f0ed194a7686934045d8807a73d6)